dev_docs/developer_scripts.md

```diff
@@ -118,3 +70,65 @@ gui-spec-windows - creates the initial spec file for windows, do not run unless
 gui-build-osx - builds the app from the osx spec file
 gui-build-windows - builds the app from the windows spec file
 ```
+## Client Makefile
+
+The following phony `make` targets in `client/Makefile` are convenience methods for getting you up and developing.
+
+### Runner scripts
+
+#### start-frontend
+
+**About** Serve the current client javascript independently from the `server` code.
+
+**Requires**
+* The server to be running. Best way to do this is with `make backend-dev`
+* `make ci` to install the necessary node modules
+
+**Usage** `make start-frontend`
+
+#### backend-dev
+
+**About** This script enables FE developers to run the REST API necessary to back the development server for the front end. It is intended to ensure that the FE developer gets the current version of the backend with a single command and no knowledge of python necessary. It creates and activates a virtual environment and installs cellxgene from the current branch.
+
+**Requires** Python3.6 - `virtual-env`, `pip`
+
+**Usage** `make backend-dev`. Optionally, you can then launch the node development server to serve the current state of the FE with `make start-frontend`. You can also select a specific dataset using `DATASET=<dataset path> make backend-dev`.
+
+**Tips** Developers will probably want to run this in parallel with the node dev server. You can either do this by running each in a separate termanial window or by running `backend-dev` in the background (add an `&` at the end of the command to run in the background: `DATASET=<dataset> make backend-dev &`).
+
+**Breakdown**
+
+| command                                  | purpose                                                   |
+| ---------------------------------------- | --------------------------------------------------------- |
+| python3.6 -m venv cellxgene              | creates cellxgene virtual environment                     |
+| source cellxgene/bin/activate            | activates virtual environment                             |
+| yes \| pip uninstall cellxgene \|\| true | uninstalls cellxgene (if installed)                       |
+| pip install -e ..                        | installs current local version of cellxgene               |
```

Comment:
is the double dot `..` correct?  I thought it was `pip install -e .`?

Reply:
True. You're right. It was incorrect in the [original](https://github.com/chanzuckerberg/cellxgene/blame/master/dev_docs/developer_scripts.md#L20). Fixing it here.